azure/packages/test/scenario-runner/testConfig_eastus.yml

```diff
@@ -0,0 +1,40 @@
+title: 10 new empty docs 20 loads eeach
+description: In this scenario we create 10 docs with no data, then just try to load them 20 times each
```

Comment:
We could probably move all these to a separate config folder.

Reply:
I made a single config file
Endpoints are now environment variables